doc/storageprofile.md

```diff
@@ -0,0 +1,198 @@
+# Storage Profiles
+
+## Introduction
+
+Storage Profile is the resource that  serves the information about recommended parameters for PVC - right now the accessMode and volumeMode.
+This can be used by CDI controllers when creating a PVC for DV. That way the DataVolume can be simplified and if the properties are missing,
+defaults can be applied from the StorageProfile.
+
+CDI provides a collection of Storage Profiles with default recommended values for some well known backends. 
+If the storage provisioner defined in storage class does not have defaults configured in CDI the resulting StorageProfile 
+has empty `claimPropertySets`.
+
+Storage profile maps 1:1 to StorageClass.
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata: 
+  name: sample-storage-profile
+spec: 
+  claimPropertySets: 
+  - accessModes:
+    - ReadWriteMany
+    volumeMode: 
+      Filesystem
+status:
+    storageClass: <string>
```

Comment:
storageClass == name right?  So this should be:
`storageClass: sample-storage-profile`

Maybe you should change this example to use a more likely storage class name.

Reply:
changed to hpp